packages/typespec-azure-resource-manager/lib/operations.tsp

```diff
@@ -108,9 +108,32 @@ op ArmResourceRead<
   ErrorResponse
 >;
 
+/**
+ * A resource HEAD operation
+ * @template Resource the resource being checked
+ * @template BaseParameters Optional. Allows overriding the operation parameters
+ * @template Parameters Optional. Additional parameters after the path parameters
+ * @template Response Optional. The success response for the read operation
+ * @template Error Optional. The error response, if non-standard.
+ */
+@autoRoute
+@doc("Check a {name} via HEAD operation", Resource)
```

Comment:
```suggestion
@doc("Check for the existence of a {name}.", Resource)
```

Since this will be user-facing docs, customers likely don't need to see the operation verb in the description